src/main/java/frc/robot/Constants.java

```diff
@@ -17,6 +17,7 @@
     public static final int intakeMotorID = 12;
     public static final int leftFlywheelID = 5;
     public static final int rightFlywheelID = 6;
+    public static final int hoodID = 7;
```

Comment:
Can we change all the variables in Constants.java to use the THIS_IS_MY_VARIABLE_NAME convention?